converted-ostack-docs/ceilometer-xml/admin-telemetry-best-practices.xml

```diff
@@ -30,7 +30,7 @@
       <step>
         <para>If polling many resources or at a high frequency, you can add additional
                         central and compute agents as necessary. The agents are designed to scale
-                        horizontally. For more information refer to the <link xlink:href="https://docs.openstack.org/ha-guide/controller-ha-telemetry.html">high availability guide</link>.</para>
+                        horizontally.</para>
```

Comment:
Just a note @dmpop - there's no replacement for the HA guide yet. So this has to be removed until a new guide is written :)